site/content/entry/yuzu-progress-report-may-2021/index.md

```diff
@@ -0,0 +1,356 @@
+
+toastUnlimited also gave us the option to [apply settings while games are running](https://github.com/yuzu-emu/yuzu/pull/6346). 
+This can help more comfortably test for ideal settings while playing.
+Keep in mind that settings that require a reboot to take effect won't be affected by the apply button.
+
+{{< imgs
+	"./apply.png| "
+  >}}
+
+Continuing with the per-game setting improvements, toast also added an exception for homebrew.
+Since most homebrew use a game-ID of 0000000000000000, using the ID to save the per-game settings would have been useless, so instead, 
+[yuzu will use the name of the homebrew file.](https://github.com/yuzu-emu/yuzu/pull/6361)
+
+And finally, toast, continuing work from german77 and [Tobi](https://github.com/FearlessTobi), [added a button to reset yuzu settings to default](https://github.com/yuzu-emu/yuzu/pull/6362). 
+
+{{< imgs
+	"./reset.png| Simple as that"
+  >}}
+
+This is sometimes needed in very rare cases, for example, when being unable to boot `Super Smash Bros. Ultimate`.
+
+## [Shake it, baby](https://youtu.be/Iac0T6dnuzM)
+
+german77 has been working on [implementing SDL motion](https://github.com/yuzu-emu/yuzu/pull/6244), with the purpose of removing our dependence on third party programs to get 
+motion input from valid controllers like Dual Sense, Joy-Cons, or Pro Controllers.
+With this PR, users can simply pair their controllers via Bluetooth and yuzu will pick them up as a valid input device, auto-mapping and enabling motion support, effectively 
+removing the need to use betterjoy, ds4windows or other third party software in yuzu.
+
+A separate PR adds [detection of two separate Joy-Cons as a single input device](https://github.com/yuzu-emu/yuzu/pull/6318).
+
+And to provide motion support on the Linux side, [v1993](https://github.com/v1993) [enables the use of HIDAPI within SDL2](https://github.com/yuzu-emu/yuzu/pull/6293). 
+
+Work continues to fine tune how the sensor data is interpreted, specially from non-Nintendo devices.
+For example, PS3 controllers operate motion on a single Z axis, but inform the remaining 2 axis to the operating system with useless NaN values, 
+making any motion calculations useless.
+[By replacing the X and Y axis values with zeros](https://github.com/yuzu-emu/yuzu/pull/6310), german77 solved the issue.
+
+Games don’t always check for obvious physical limitations. 
+For example in the past it was possible to use handheld control emulation while the game was in a docked status.
+Doing this can break games like `Xenoblade Chronicles 2`, as the game expects the console to disable input when the Joy-Cons are connected to a docked Switch’s rails.
+german77 solves this by [forcing an emulated Pro Controller instead](https://github.com/yuzu-emu/yuzu/pull/6353). 
+No more weird Frankenstein input monstrosities, please.
+
+Finally, yuzu could be hard to map on analog sticks due to a bug in how the event was handled. While fixing this, german77 also 
```

Comment:
```suggestion
Finally, it could sometimes be difficult to map analog sticks in yuzu due to a bug in how the event was handled. While fixing this, german77 also 
```